exec/pandora2eager.R

```diff
@@ -9,33 +9,53 @@ library(readr)
 library(tidyr)
 library(stringr)
 library(pandora2eager)
+library(optparse)
+
+# Function to validate the file type
+validate_file_type <- function(option, opt_str, value, parser) {
+  valid_entries <- c("bam","fastq_pathogens") ## TODO comment: should this be embedded within the function? You would want to maybe update this over time no? 
```

Comment:
Indeed, the valid entries could be passed in as an argument to the function. Feel free to make the change if you want.

Reply:
I think since it does not really matter, I will leave it within the function